repository-data/application/src/main/resources/hcm-content/content/documents/administration/labels/pages/listicle.yaml

```
@@ -14,25 +14,28 @@
     resourcebundle:messages_de: ['Die folgende Liste wurde von VisitScotland erstellt,
         um eine Auswahl an Erlebnissen zu präsentieren. Sie sind in keiner bestimmten
-        Reihenfolge aufgelistet.']
+        Reihenfolge aufgelistet.', Ausstattung]
     resourcebundle:messages_es: [La siguiente lista ha sido elaborada por VisitScotland
         para mostrar una amplia gama de experiencias. No responde a un orden o clasificación
-        establecidos.]
+        establecidos., Instalaciones destacadas]
     resourcebundle:messages_fr: [La liste suivante a été créée par VisitScotland afin
-        de présenter différentes expériences et n’est classée dans aucun ordre particulier.]
+        de présenter différentes expériences et n’est classée dans aucun ordre particulier.,
+      Infrastructures]
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 33 but found 6

(indentation)